packages/js/images/connection-assistant.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 752 247" stroke="#000" stroke-linecap="round"
```

Comment:
Now that I noticed the other one, this SVG is also missing the normal `role="img" aria-hidden="true" focusable="false"` attributes